### PR TITLE
Initial removal of manage_by_attributes

### PR DESCRIPTION
**Pull request by pasimko**
Did not see any tests that needed changing

### CODE REVIEW

**Review by ramereth — CHANGES_REQUESTED** [review]

File: spec/unit/recipes/packer_pipeline_node_spec.rb [hunk 1]
```diff
@@ -79,12 +79,13 @@
...
-      #   expect(chef_run).to upgrade_chef_gem('openstack_taster').with(
-      #     options: '--no-user-install',
-      #     version: '>= 2.0',
-      #     clear_sources: true
-      #   )
+      #  expect(chef_run).to upgrade_chef_gem('openstack_taster').with(
+      #    options: '--no-user-install',
+      #    version: '>= 2.0',
+      #    clear_sources: true
+      #  )
```
Comment (ramereth) [inline spec/unit/recipes/packer_pipeline_node_spec.rb:88]: Please go ahead and remove this.

File: test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb [hunk 2]
```diff
@@ -46,10 +46,10 @@
...
 end
 
-describe file("/opt/#{chef}/embedded/bin/openstack_taster") do
-  it { should be_a_file }
-  it { should be_executable }
-end
+# describe file("/opt/#{chef}/embedded/bin/openstack_taster") do
+#  it { should be_a_file }
+#  it { should be_executable }
+# end
```
Comment (ramereth) [inline test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb:52]: Please just remove this (and above) instead of commenting it out.

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.16.1.  Have a nice day!

**Pull request merged**